src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -33,17 +36,25 @@ const useForm = (initialValues, onSubmit) => {
     handleDegree: (event) => {
       update('degree')(event.target.value);
     },
+    handleEndDateNone: (e) => {
+      e.persist();
+      values.is_current = e.target.checked;
+      if (isEndDateNull) {
```

Comment:
Looks like a bug here, this is a function but you're not calling it (so this will always be true, since function types are truthy values)

Also, it's not super clear to me that calling this function is better than just checking `values.is_current === true`, since we only really care about `is_current` here, not the other possibilities checked within isEndDateNull

As a third note, this function's name is misleading (on reading it, I thought it checked 'values' to see if `end_month === 'none'`, etc.) I would suggest the name `shouldEndDateBeNulled` or `shouldClearEndDate` instead

Reply:
done